apps/activityreminder/settings.js

```diff
@@ -75,7 +75,14 @@
           settings.tempThreshold = v;
           activityreminder.writeSettings(settings);
         }
-      }
+      },
+      'Unlock on alarm': {
+        value: settings.unlock,
```

Comment:
You might want `value: !!settings.unlock,` here?

If `unlock` wasn't previously set (an old install?) this won't know it's meant to be a bool so won't be editable